core/src/Kokkos_ExecPolicy.hpp

```diff
@@ -906,5 +906,40 @@ require(const PolicyType p, WorkItemProperty::ImplWorkItemProperty<P>) {
       WorkItemProperty::ImplWorkItemProperty<P>, PolicyType>::policy_out_t(p);
 }
 }  // namespace Experimental
+
+namespace Impl {
+
+template <class PatternTag, class... Args>
+struct PatternImplSpecializationForTag;
```

Comment:
I would like From instead of For for the names.

Reply:
addressed in 7440d02